docs/sources/flow/reference/components/otelcol.receiver.opencensus.md

```diff
@@ -0,0 +1,234 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.receiver.opencensus
+title: otelcol.receiver.opencensus
+---
+
+# otelcol.receiver.opencensus
+
+`otelcol.receiver.opencensus` accepts telemetry data via gRPC or HTTP 
+using the [OpenCensus](https://opencensus.io/) format and
+forwards it to other `otelcol.*` components.
+
+> **NOTE**: `otelcol.receiver.opencensus` is a wrapper over the upstream
+> OpenTelemetry Collector `opencensus` receiver from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.receiver.opencensus` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.receiver.opencensus "LABEL" {
+  cors_allowed_origins = [ ... ]
+
+  grpc { ... }
+
+  output {
+    metrics = [...]
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+In order to use plain HTTP/JSON, specify the `endpoint` attribute
+inside the `grpc` block of this component. There is no HTTP-specific block.
+The HTTP/JSON address is the same as gRPC as the protocol is recognized and processed accordingly.
+
+To write traces with HTTP/JSON, `POST` to `[address]/v1/trace`. The JSON message format parallels the gRPC protobuf format, see this [OpenApi spec for it](https://github.com/census-instrumentation/opencensus-proto/blob/master/gen-openapi/opencensus/proto/agent/trace/v1/trace_service.swagger.json).
+
+## Arguments
+
+`otelcol.receiver.opencensus` supports the following arguments:
+
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`cors_allowed_origins`     | `list(string)` | A list of allowed Cross-Origin Resource Sharing (CORS) origins. |  | no
+
+`cors_allowed_origins` are the allowed CORS origins for HTTP/JSON requests to gRPC-gateway adapter 
+for the OpenCensus receiver. See [github.com/rs/cors](github.com/rs/cors).
```

Comment:
I don't know if anyone will know what "gRPC-gateway adapter" means :)

Reply:
Haha fair enough, I'll leave it as:

> `cors_allowed_origins` are the allowed [CORS](github.com/rs/cors) origins for HTTP/JSON requests.